### PR TITLE
[Feature] Toast to DS-site

### PR DESCRIPTION
**Pull request by valger11**
- Update to use suomifi-ui-components v6.2.0-beta.2
- Add Toast component page

### CODE REVIEW

File: locale/fi/toast.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "Sivulla voi olla yhtä aikaa useita Toast-ilmoituksia näkyvissä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Komponentti koostuu kuvaavasta ikonista, otsikosta ja pääsisällöstä. Komponentissa ei ole eri tiloja (status variant)."
```
Comment (LJKaski) [inline locale/fi/toast.json:17]: I don't think the lack of status variants needs to be stated.

File: locale/fi/toast.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ilmoitus (Toast)",
+  "intro": "Ilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää tietoa käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Toast-ilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
```
Comment (ketsappi) [inline locale/fi/toast.json:3]: > .. voidaan välittää **tietoa** käyttäjän toimintaan liittyvää palautetta..

Unwanted text, "tietoa". Should remove in my opinion.

File: locale/fi/toast.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      ],
+      "links": [
+        {
+          "text": "Live region",
+          "url": "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions"
+        }
+      ]
+    }
+  ],
+  "exampleRegular.content": "Tallentaminen onnistui"
```
Comment (ketsappi) [inline locale/fi/toast.json:49]: Maybe the key could be simplified as there is no possibility to conflict with different kind of examples here.
Maybe just `example.content` (changing it here and in other .json files of course)

File: locale/af/toast.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (ketsappi) [inline locale/af/toast.json:1]: Look example how the so called translation source file (af language) is filled in other components.

Key and actual value should be same for it to be in line with others.
E.g `"title": "",` --> `"title": "title",`

File: src/pages/components/toast.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import { Toast } from 'components/ExampleComponents';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+import Section from 'components/Section';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentExample from 'components/ComponentExample';
+import { defaultSuomifiTheme } from 'suomifi-ui-components';
```
Comment (ketsappi) [inline src/pages/components/toast.tsx:13]: This import is better to be located after the import of React. First external imports and then internal ones.

File: locale/fi/toast.json (hunk 1 above)
Comment (valger11) [reply locale/fi/toast.json]: Fixed.

File: locale/fi/toast.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Toast-ilmoitukset pysyvät näkyvissä joko 10 sekunttia tai niin kauan kuin käyttäjä pitää hiirtä ilmoituksen päällä. Ilmoitukset poistuvat näkyvistä liukumalla ylös."
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Desktopissa, kun sivu on scrollattu ylös, ilmoitus näkyy sivun ylälaitaan kiinnittyneenä. Kun käyttäjä scrollaa sivulla alaspäin niin, että sivu ylälaita peittyy, liukuu ilmoitus näkyviin oikeaan ylälaitaan. Ilmoitus siis kelluu mukana, kun sivua scrollataan alas."
```
[the remark this answers is not in the document]
Comment (valger11) [reply locale/fi/toast.json]: Fixed translations.

File: locale/fi/toast.json (hunk 2 above)
Comment (valger11) [reply locale/fi/toast.json]: Removed it.

File: locale/fi/toast.json (hunk 3 above)
Comment (valger11) [reply locale/fi/toast.json]: changed to "example.content"

File: locale/af/toast.json (hunk 4 above)
Comment (valger11) [reply locale/af/toast.json:1]: This was fixed before.

File: src/pages/components/toast.tsx (hunk 5 above)
Comment (valger11) [reply src/pages/components/toast.tsx]: Fixed place of imports

File: locale/fi/toast.json (hunk 1 above)
Comment (LJKaski) [reply locale/fi/toast.json]: Great. One more thought for the same line that occurred to me: is the word "kuvaavasta" really necessary there? Also, should we mention that the heading is optional?

File: locale/fi/toast.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (LJKaski) [reply locale/fi/toast.json]: Looking better! There still seems to be the word scrolling used twice though.

File: locale/fi/toast.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ponnahdusilmoitus (Toast)",
+  "intro": "Ponnahdusilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Ponnahdusilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ponnahdusilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ponnahdusilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ponnahdusilmoituksesta.",
```
Comment (LJKaski) [inline locale/fi/toast.json:6]: I'm still not quite sure if this is clear enough. Somehow when I read this line, I get the impression that the live region itself is an external link at the bottom of the page. Not sure if we should omit the link entirely and just assume they can look up what live region means. What do you think?

File: locale/fi/toast.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    "Ponnahdusilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan.",
+    "Sivulla voi olla yhtä aikaa useita ponnahdusilmoituksi näkyvissä."
```
Comment (LJKaski) [inline locale/fi/toast.json:8]: "ponnahdusilmoituksi" is missing an "a" at the end.

File: locale/fi/toast.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ponnahdusilmoitus (Toast)",
+  "intro": "Ponnahdusilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Ponnahdusilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ponnahdusilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ponnahdusilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ponnahdusilmoituksesta.",
+    "Ponnahdusilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan.",
```
Comment (LJKaski) [inline locale/fi/toast.json:7]: What are the different use cases that require different heading levels? Is this relevant information? In the ingress it is mentioned that the toast is specifically for success messages, so what would the different use cases be? It's a bit confusing.

File: locale/fi/toast.json (hunk 8 above)
Comment (valger11) [reply locale/fi/toast.json]: fixed

File: locale/fi/toast.json (hunk 7 above)
Comment (valger11) [reply locale/fi/toast.json]: fixed to be:
"(kts. linkki sivun alalaidassa)"

File: locale/fi/toast.json (hunk 9 above)
Comment (valger11) [reply locale/fi/toast.json]: Fixed it to be: "Ponnahdusilmoituksen otsikkotaso on määriteltävissä tarvittaessa, ja komponenttia voidaan käyttää myös ilman otsikkoa."

File: locale/fi/toast.json (hunk 3 above)
Comment (ketsappi) [reply locale/fi/toast.json]: Could you change the value in the **af/toast.json** also, now it has the old keyname in it.
https://github.com/vrk-kpa/suomifi-design-system/blob/0bc89c22d852c7e3b32b69016680df7e6ab1981e/locale/af/toast.json#L45

File: locale/af/toast.json (hunk 4 above)
Comment (ketsappi) [reply locale/af/toast.json:1]: Value should not be in plural, so just `section.title` instead. _This is twice in the same file._

https://github.com/vrk-kpa/suomifi-design-system/blob/0bc89c22d852c7e3b32b69016680df7e6ab1981e/locale/af/toast.json#L8

File: locale/fi/toast.json (hunk 1 above)
Comment (valger11) [reply locale/fi/toast.json]: fixed.

File: locale/fi/toast.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (valger11) [reply locale/fi/toast.json]: fixed.

File: locale/fi/toast.json (hunk 3 above)
Comment (valger11) [reply locale/fi/toast.json]: fixed.

File: locale/af/toast.json (hunk 4 above)
Comment (valger11) [reply locale/af/toast.json:1]: fixed

File: locale/fi/toast.json [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+        }
+      ]
+    },
+    {
+      "title": "Koko ja käyttö",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Ponnahdusilmoitukset tulee pitää näkyvissä joko 10 sekunttia tai niin kauan kuin käyttäjä pitää hiirtä ilmoituksen päällä. Ponnahdusilmoitukset tulee poistaa näkyvistä liukumalla ylös."
```
[the remark this answers is not in the document]
Comment (valger11) [reply locale/fi/toast.json:33]: fixed

**Review by ketsappi — APPROVED** [review]

File: locale/fi/toast.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (LJKaski) [reply locale/fi/toast.json]: Did you do something about the last sentence I asked about? Looks the same. I'd love to hear your thoughts.

[the remark this answers is not in the document]
Comment (LJKaski) [reply locale/fi/toast.json]: Looks better now 👍

**Review by LJKaski — APPROVED** [review]

**Pull request merged**